hellogsm-web/src/main/java/team/themoment/hellogsm/web/domain/application/repository/ApplicationRepository.java

```diff
@@ -26,18 +26,72 @@ public interface ApplicationRepository extends JpaRepository<Application, Long>
 
     Page<Application> findAll(Pageable pageable);
 
-    Page<Application> findAllByAdmissionInfoApplicantNameContainingAndAdmissionStatus_IsFinalSubmitted(String keyword, Boolean isFinalSubmitted, Pageable pageable);
-
-    Page<Application> findAllByAdmissionInfoSchoolNameContainingAndAdmissionStatus_IsFinalSubmitted(String keyword, Boolean isFinalSubmitted, Pageable pageable);
-
-    @Query("SELECT a FROM Application a WHERE " +
+    @Query("SELECT a FROM Application a " +
+            "WHERE a.admissionInfo.applicantName LIKE %:keyword% " +
+            "AND a.admissionStatus.isFinalSubmitted = :isFinalSubmitted " +
+            "ORDER BY " +
+            "CASE " +
+            "WHEN a.admissionStatus.secondEvaluation='PASS' THEN 30 " +
+            "WHEN a.admissionStatus.secondEvaluation='FALL' THEN 20 " +
+            "ELSE 10" +
+            "END DESC, " +
+            "CASE " +
+            "WHEN a.admissionStatus.firstEvaluation='PASS' THEN 3 " +
+            "WHEN a.admissionStatus.firstEvaluation='FALL' THEN 2 " +
+            "ELSE 1" +
+            "END DESC")
+    Page<Application> findAllByIsFinalSubmittedAndApplicantNameContaining(@Param("keyword") String keyword, @Param("isFinalSubmitted") Boolean isFinalSubmitted, Pageable pageable);
+
+    @Query("SELECT a FROM Application a " +
+            "WHERE a.admissionInfo.schoolName LIKE %:keyword% " +
+            "AND a.admissionStatus.isFinalSubmitted = :isFinalSubmitted " +
+            "ORDER BY " +
+            "CASE " +
+            "WHEN a.admissionStatus.secondEvaluation='PASS' THEN 30 " +
+            "WHEN a.admissionStatus.secondEvaluation='FALL' THEN 20 " +
+            "ELSE 10" +
+            "END DESC, " +
+            "CASE " +
+            "WHEN a.admissionStatus.firstEvaluation='PASS' THEN 3 " +
+            "WHEN a.admissionStatus.firstEvaluation='FALL' THEN 2 " +
+            "ELSE 1" +
+            "END DESC")
```

Comment:
가독성 좋게 들여쓰기를 추가하면 좋을 것 같아요.
나머지 부분도 수정했으면 좋겠습니다.
```suggestion
 @Query("SELECT a FROM Application a " +
       "WHERE a.admissionInfo.schoolName LIKE %:keyword% " +
       "AND a.admissionStatus.isFinalSubmitted = :isFinalSubmitted " +
       "ORDER BY " +
       "  CASE " +
       "    WHEN a.admissionStatus.secondEvaluation = 'PASS' THEN 30 " +
       "    WHEN a.admissionStatus.secondEvaluation = 'FALL' THEN 20 " +
       "    ELSE 10 " +
       "  END DESC, " +
       "  CASE " +
       "    WHEN a.admissionStatus.firstEvaluation = 'PASS' THEN 3 " +
       "    WHEN a.admissionStatus.firstEvaluation = 'FALL' THEN 2 " +
       "    ELSE 1 " +
       "  END DESC")
```

Reply:
수정하였습니다